### ISSUE TITLE
Cannot focus send, receive buttons with VoiceOver screen reader

### ISSUE THREAD

**Issue by codeofdusk** [horizontalsystems/unstoppable-wallet-ios#3747]
## STR:
1. Enable VoiceOver.
1. Start Unstoppable.
1. Find (either by exploring the screen or with swipe gestures) the "balance" tab.
1. Search the screen (either with swipe gestures or exploration) for the "send" or "receive" buttons.

## Expected behaviour:
The buttons are focusable and can be activated.

## Observed behaviour:
VoiceOver is unable to see the buttons.

## Additional context:
I'm a VoiceOver user (totally blind) and this makes the app difficult/impossible to use for me.

See [Supporting VoiceOver in Your App](https://developer.apple.com/documentation/accessibility/supporting_voiceover_in_your_app) in the Apple Developer docs.

**Comment by esengulov** [comment]
thanks for the input, we will look into it.

**Comment by esengulov** [comment]
https://github.com/horizontalsystems/unstoppable-wallet-ios/issues/4228

**Comment by esengulov** [comment]
hi @codeofdusk we have evaluated this task and it seems that there is some singifcant amount of work needs to be done on our part to have this implemented properly. Are you still looking for the wallet? We are keeping this issue open.

**Comment by codeofdusk** [comment]
Yes, this issue is still present and a major blocker.

**Comment by Dianaaiym** [comment]
Closing as part of issue cleanup
